gatsby-node.js

```diff
@@ -60,10 +63,10 @@ exports.createPages = async ({ graphql, actions }) => {
       query {
         allPagesJson(
           filter: { investigation: { eq: "exploding-stars" } }
-          sort: { fields: [sectionId, order, investigation], order: ASC }
+          sort: { fields: [sectionOrder, order, investigation], order: ASC }
```

Comment:
"sectionId" has been replaced with a simple "sectionOrder" which then maps to an array of available sections.